### PR TITLE
Add MAV_CMD_DO_SPRAYER to common.xml (was in ardupilot)

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -1311,6 +1311,16 @@
         <param index="1" label="System ID" minValue="1" maxValue="255" increment="1">System ID</param>
         <param index="2" label="Gimbal device ID">Component ID of gimbal device to address (or 1-6 for non-MAVLink gimbal), 0 for all gimbal device components. Send command multiple times for more than one gimbal (but not all gimbals).</param>
       </entry>
+      <entry value="199" name="MAV_CMD_DO_SPRAYER" hasLocation="false" isDestination="false">
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml:1314]: Any reason this couldn't be proposed separately upstream? I'll add some questions

Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml:1314]: It is being proposed here. :)

Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml:1314]: How long will it take this to get discussed?

Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml:1314]: I moved the other commit out of this PR and will merge that other one in. So let's start the discussion/review now.

Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml:1314]: nudge

**Comment by amilcarlucas** [comment]
If no one complains I will merge it in the next few hours. Thanks.

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -1311,6 +1311,16 @@
...
+        <description>Control attached liquid sprayer</description>
+        <param index="1" label="Sprayer Enable">Sprayer Enable.</param>
```
Comment (julianoes) [inline message_definitions/v1.0/common.xml:1316]: So 1 to enable and 0 to disable?

Comment (amilcarlucas) [reply message_definitions/v1.0/common.xml:1316]: yes.

Comment (julianoes) [reply message_definitions/v1.0/common.xml:1316]: I would spell that out in the comment, then I'm good with this.

**Review by julianoes — COMMENTED** [review]
Great, thanks @amilcarlucas!

**Pull request merged**